packages/geo-diary/src/components/ui/map/google-map-component.tsx

```diff
@@ -381,9 +384,18 @@ export const GoogleMapComponent: FC<MapProps> = ({ appointments }) => {
           onLoaded={handleOnLoaded(mapRefs)}
           zoom={DEFAULT_ZOOM}
           center={{ lat: currentLat ?? DEFAULT_LAT_LNG.lat, lng: currentLng ?? DEFAULT_LAT_LNG.lng }}
+          mapTypeControl={false}
+          fullscreenControl={false}
+          zoomControlOptions={{
+            position: google?.maps?.ControlPosition.TOP_RIGHT,
+          }}
+          streetViewControlOptions={{
+            position: google?.maps?.ControlPosition.TOP_RIGHT,
+          }}
         />
+        {(isMobile || isTablet) && routeInformation && <TravelMode />}
```

Comment:
Don't know if this is worth bringing up but do you think this condition should be reversed? `routeInformation && (isMobile || isTablet)`? Just thinking about what would likely be true or false to determine